receiver.go

```diff
@@ -265,7 +266,9 @@ func (r *receiver) handleMessage(ctx context.Context, msg *amqp.Message, handler
 
 	err = handler(ctx, event)
 	if err != nil {
-		err = r.receiver.ModifyMessage(ctx, msg, true, false, nil)
+		err = r.receiver.ModifyMessage(ctx, msg, &amqp.ModifyMessageOptions{
```

Comment:
Doesn't have to change, but I think these settlement calls are essentially no-ops since we're in ModeFirst.